benchmarks/benchmarks/sparse_linalg_lobpcg.py

```diff
@@ -4,39 +4,41 @@
 from .common import Benchmark, safe_import
 
 with safe_import():
-    from scipy import array, r_, ones, arange, sort, diag, cos, rand, pi
-    from scipy.linalg import eigh, orth, cho_factor, cho_solve
+    from scipy.linalg import eigh, cho_factor, cho_solve
     import scipy.sparse
-    from scipy.sparse.linalg import lobpcg
+    from scipy.sparse.linalg import lobpcg, eigsh
     from scipy.sparse.linalg._interface import LinearOperator
 
 
-def _sakurai(n):
-    """ Example taken from
+def _sakurai_rev(n):
+    """ Example turns a generalized eigenproblem for the matrix pair A and B
         T. Sakurai, H. Tadano, Y. Inadomi and U. Nagashima
         A moment-based method for large-scale generalized eigenvalue problems
```

Comment:
Can you wrap the docstring correctly please?
```suggestion
    """
    Example turns a generalized eigenproblem for the matrix pair A and B
    T. Sakurai, H. Tadano, Y. Inadomi and U. Nagashima
    A moment-based method for large-scale generalized eigenvalue problems
```